includes/Story_Post_Type.php

```diff
@@ -913,4 +915,45 @@ public function change_default_title( $data ) {
 		}
 		return $data;
 	}
+
+	/**
+	 * Add VTT file type to allow file in multisite.
+	 *
+	 * @param string $value List of allowed file types.
+	 * @return string List of allowed file types.
+	 */
+	public function add_tracks_files_ms( $value ) {
+		$filetypes = explode( ' ', $value );
+		if ( ! in_array( 'vtt', $filetypes, true ) ) {
+			$filetypes[] = 'vtt';
+			$value       = implode( ' ', $filetypes );
+		}
+
+		return $value;
+	}
+
+	/**
+	 * Get a list of available.
+	 *
+	 * @return array Array of languages.
+	 */
+	protected function get_languages() {
+		if ( ! function_exists( 'wp_get_available_translations' ) ) {
+			require_once ABSPATH . 'wp-admin/includes/translation-install.php';
+		}
+		$languages              = [];
+		$available_translations = wp_get_available_translations();
+		foreach ( $available_translations as $slug => $available_translation ) {
+			$language = $available_translation['native_name'];
+			if ( $available_translation['native_name'] !== $available_translation['english_name'] ) {
+				$language = sprintf( '%s (%s)', $available_translation['native_name'], $available_translation['english_name'] );
+			}
+			$languages[] = [
+				'name'  => $language,
+				'value' => $slug,
```

Comment:
We can't simply use the slug here, as the locale slug used by WordPress doesn't have the same structure as the required one by HTML.

As per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/track, `srclang` must be a valid [BCP 47 language tag](https://r12a.github.io/app-subtags/).

BCP 47 tag for German: `de`

What WP provides in `wp_get_available_translations()`: `de_DE`, `de_DE_formal`, `de_CH`, `de_CH_informal`.

So we cannot use the locales provided by WordPress as-is.

Reply:
```suggestion
				'value' => current( $available_translation['iso'] ),
```

That should work for now. The downside is it will result in `de` being added no matter if you use "German (Germany)" or "German (Switzerland)", but that should be OK for now

Reply:
This isn't going to work. That would result in multiple values of the same. Meaning the correct value would not be set on the dropdown. It would just pick the first one item that matched. In english case, it would be `en_CA`. I think we might need a fixed list.